steps/step-jira-number-from-branch/1.0.0/step.yml

```diff
@@ -0,0 +1,66 @@
+title: jira-number-from-branch
```

Comment:
Please use a more UX friendly title here. This is the name of the step the users will see int eh workflow editors step selector panel.